apps/monday-code-action-template-node/.gitignore

```diff
@@ -1,3 +1,5 @@
+.env
```

Comment:
why? usually we aren't ignoring this

Reply:
Because it holds secrets and stuff, it felt wrong to add it